EIPS/eip-5585.md

```diff
@@ -0,0 +1,111 @@
+---
+eip: 5585
+title: EIP-721 NFT Authorization
+description: NFT owners can authorize other users to use their NFTs for business purpose.
+author: Veega Labs (@VeegaLabsOfficial)
+discussions-to: https://ethereum-magicians.org/t/nft-authorization-erc721-extension/10661
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-15
+requires: 721
+---
+
+## Abstract
+
+1. By registering and confirming ownership and other commercial rights independently on the chain, it supports the independent circulation of commercial authorization. Thereby the users will get more benefits in addition to selling, and it will significantly motivate users in various ways.
+2. After separating the ownership of assets and other commercial rights, the information of full authorization chain will be saved by smart contracts, and the benefits of commercial users will be protected by smart contracts on the blockchain, and would not be affected by the changes of ownership.
+3. The cryptocurrency paid by the users will be hosted and locked in the smart contract, and the "authorized funds" will be dynamically unlocked to asset owners based on how long the assets being used. （this feature will be implemented in the future）
```

Comment:
This should be a paragraph, not a list. It should also describe a high-level overview of the solution described in the spec.